src/javascript/components/SitemapPanelApp.jsx

```diff
@@ -149,82 +155,91 @@ const SitemapPanelApp = ({client, t}) => {
+                                    <SitemapIndexLink
+                                        inputUrl={formik.values.sitemapIndexURL}
+                                        siteKey={currentState.site}
+                                        t={t}/>
+                                </div>
+                                <div className={styles.subsection}>
+                                    <Typography className={styles.updateIntervalTitle} component="h3">
+                                        {t('labels.settingSection.updateIntervalSection.title')}
+                                    </Typography>
+                                    <Typography className={styles.updateIntervalDescription} component="p">
+                                        {t('labels.settingSection.updateIntervalSection.description')}
+                                    </Typography>
+
+                                    <Dropdown
+                                        id="intervalDuration"
+                                        data-sel-role="sitemapIntervalDuration"
+                                        name="intervalDuration"
+                                        isDisabled={false}
+                                        variant="outlined"
+                                        label={convertCacheDurationToLabel(formik.values.sitemapCacheDuration)}
+                                        value={formik.values.sitemapCacheDuration}
+                                        data={dropdownData}
+                                        onChange={(e, item) => {
+                                            formik.setFieldValue('sitemapCacheDuration', item.value);
+                                        }}
+                                    />
+                                </div>
+                            </section>
                         </div>
-                    }
-                    handleClose={handleSnackBarClose}
-                />
-            </main>
-        </form>
+                    </div>
+                    <SnackbarComponent
+                        isOpen={snackbarIsOpen}
+                        autoHideDuration={10000}
+                        message={
+                            <div className={styles.snackbarMessageDiv}>
+                                {snackbarInfo && snackbarInfo.error ?
+                                    <Close className={styles.snackbarErrorCheckIcon}/> :
+                                    <Check className={styles.snackbarMessageCheckIcon}/>}
+                                <Typography className={styles.snackbarMessageTypography} component="p">
+                                    {snackbarInfo && snackbarInfo.message}
+                                </Typography>
+                            </div>
+                        }
+                        handleClose={handleSnackBarClose}
+                    />
+                </main>
+            </form>
+        </MuiThemeProvider>
```

Comment:
> Optional: Not related to your PR but we could have use the react hooks useTranslation and useApollo instead of compose for this component
I would prefer to not change the current implementation.